### PR TITLE
Update ANTICHEATS.md

### PR DESCRIPTION
**Pull request by steviebeenz**
Update info on my own products.

# Description

What did you add? or change?
Update info on my own products.

## Checklist

- [x] My contributions follows the style guidelines of this project
- [x] I have performed a self-review of my own contributions (checked for spelling errors, proper linking, etc)
- [ ] I have run linting on my branch before submitting the PR

### CODE REVIEW

**Comment by steviebeenz** [comment]
WIP, smfn seems to have caused it to break another Anticheat's link.

**Comment by JustDoom** [comment]
@steviebeenz What link broke?

**Comment by steviebeenz** [comment]
> @steviebeenz What link broke?

ZK Anticheat due to my autocorrect

File: server/info/lists/plugins/ANTICHEATS.md [hunk 1]
```diff
@@ -760,9 +760,9 @@ Rating: 3.563
 Discontinued
 - [zHack Lite](https://polymart.org/resource/460)  
 Version: 1.8.8  
-Rating: ???  
+Rating: 5
 Currently Active
-- [ZK Anticheat](https://github.com/Valou447/ZK-ANTICHEAT)  
+- [ZK Anticheat](https://github.comlou447/ZK-ANTICHEAT)
```
Comment (JustDoom) [inline server/info/lists/plugins/ANTICHEATS.md:765]: press comit
```suggestion
- [ZK Anticheat](https://github.com/Valou447/ZK-ANTICHEAT)  
```

**Pull request merged**